src/MPS.jl

```diff
@@ -38,6 +38,21 @@ Base.zero(x::T) where {T<:Union{MPS,MPO}} = T(zero(Ansatz(x)), form(x))
 defaultorder(::Type{<:AbstractMPS}) = (:o, :l, :r)
 defaultorder(::Type{<:AbstractMPO}) = (:o, :i, :l, :r)
 
+function MPS(
+    arrays::Vector{<:AbstractArray}; order=defaultorder(MPS), form::Form=NonCanonical(), check_canonical_form=true
+)
+    return MPS(form, arrays; order=order, check_canonical_form=check_canonical_form)
+end
+function MPS(
+    arrays::Vector{<:AbstractArray},
+    λ::Vector{<:AbstractArray};
+    order=defaultorder(MPS),
+    form::Form=Canonical(),
+    check_canonical_form=true,
+)
+    return MPS(form, arrays, λ; order=order, check_canonical_form=check_canonical_form)
+end
+
```

Comment:
1. I would just call it `check`. `check_canonical_form` is too long and for the size checks we already use the `@unsafe_region`.
2. If the variable you're passing to the kwarg is named the same as the kwarg, you don't to write the `=`. It's cleaner this way.
3. For default kwargs that are just passed by, use `kwargs...`
4. If they are not really needed, don't put the types in the args.
  - Types in function signatures should only (mostly) be used for dispatch

```suggestion
MPS(arrays; form::Form=NonCanonical(), kwargs...) = MPS(form, arrays; kwargs...)
function MPS(arrays, λ; form::Form=Canonical(), kwargs...)
    return MPS(form, arrays, λ; kwargs...)
end

```

Reply:
I need to add the types in the args because we need to distinguish from the `MPS(ansatz::Ansatz, form::Form)` constructor. Other than that, I agree with your suggestions!